docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -228,76 +232,32 @@ You can update to the Grafana Mimir Helm chart from the Cortex Helm chart.
    b. Use `mimirtool` to update the configuration.
 
    ```bash
-   mimirtool config convert --yaml-file cortex.yaml
+   mimirtool config convert --gem --yaml-file cortex.yaml --yaml-out mimir.yaml
```

Comment:
why is `--gem` needed now for cortex?

Reply:
Accidental addition. Have been working with gem migration documentation at the same time.